OrbitService/Utils.cpp

```diff
@@ -138,13 +139,31 @@ ErrorMessageOr<std::vector<ModuleInfo>> ParseMaps(std::string_view proc_maps_dat
 ErrorMessageOr<std::vector<orbit_grpc_protos::TracepointInfo>> ReadTracepoints() {
   std::vector<TracepointInfo> result;
 
-  for (const auto& category : fs::directory_iterator(kLinuxTracingEvents)) {
-    if (fs::is_directory(category)) {
-      for (const auto& name : fs::directory_iterator(category)) {
-        TracepointInfo tracepoint_info;
-        tracepoint_info.set_name(fs::path(name).filename());
-        tracepoint_info.set_category(fs::path(category).filename());
-        result.emplace_back(tracepoint_info);
+  struct stat info;
```

Comment:
That looks a bit like the 90s are back. What was wrong with the std::filesystem based solution?